### PR TITLE
Costars fix

### PR DESCRIPTION
**Pull request by hamhands**
### What changed
- Fixed costars upload bug where POST was causing hiccups, moved to a post > process > redirect flow
- Rewrote tests to match change
- Fixed costars import bug causing contract description problems (no more 'Ostars')
- Added filename convention requirement
### Screencap

Error message with filename requirement:
screen shot 2015-08-07 at 5 04 43 pm

### CODE REVIEW

File: purchasing_test/unit/import/test_costars_import.py [hunk 1]
```diff
@@ -36,3 +36,7 @@ def test_costars_import(self):
         # assert the county importer works properly
         self.assertEquals(len(props['Located in']), 2)
         self.assertEquals(len(props['List of manufacturers']), 2)
+
+    def test_raises_bad_filename(self):
+        with self.assertRaises(IOError):
+            self.assertRaises(main('', 'BADFILENAME', None, None, None))
```
Comment (bsmithgall) [inline purchasing_test/unit/import/test_costars_import.py]: redundant second `self.assertRaises`.

File: purchasing_test/unit/conductor/test_conductor_upload.py [hunk 2]
```diff
@@ -75,7 +75,8 @@ def test_upload_locked(self):
             self.login_user(user)
             test_file = self.create_file('test.csv', 'text/csv')
             req = self.client.post('/conductor/upload/costars', data=dict(upload=test_file))
-            self.assert200(req)
+            self.assertEqual(req.status_code, 302)
+            self.assertRaises(render_template('conductor/upload/upload_success.html'))
```
Comment (bsmithgall) [inline purchasing_test/unit/conductor/test_conductor_upload.py]: This doesn't seem right.

File: purchasing_test/unit/import/test_costars_import.py (hunk 1 above)
Comment (hamhands) [inline purchasing_test/unit/import/test_costars_import.py]: can you take a look now and tell me why what I did is stupid?

**Comment by bsmithgall** [comment]
It looks like there is some strangeness going on with the history -- those scout logos are already in master.

**Comment by bsmithgall** [comment]
Other than that, :+1:

**Comment by hamhands** [comment]
@bsmithgall I hate history

**Comment by bsmithgall** [comment]
@hamhands this should be fixed.

**Pull request merged**